source/MaterialXCore/Geom.h

```diff
@@ -348,22 +348,47 @@ class GeomAttr : public ValueElement
     static const string CATEGORY;
 };
 
-/// @class GeomProp
+/// @class GeomPropDef
 /// An element representing a declaration of geometric input data.
-/// Can be used as a child to Input elements to declare a geometric node
-/// to be used if no explicit connection is set.
-/// Can also be used as children to NodeDef elements to declare internal
-/// geometric requirements for a node. That is, required geometric input
-/// data that are not published as ordinary user editable inputs.
-class GeomProp : public TypedElement
+/// Can be used to declare a geometric property with specific modifiers set,
+/// and assing this a name to be referenced elsewhere in a document. 
```

Comment:
typo. Not sure what "assing" means here :) ?

Reply:
Lol, I'll fix this :)